src/pluto/pluto.py

```diff
@@ -521,45 +521,39 @@ def create_outlier(df, v1, v2, condo, mapped):
         outlier_records = outlier.to_dict("records")
         v1_outlier = [i["outlier"] for i in outlier_records if i["v"] == v1][0]
 
-        def display_dataframe(v1_outlier, field):
+        def fetch_dataframe(v1_outlier, field):
             records = [i["values"] for i in v1_outlier if i["field"] == field][0]
             if records:
                 df = pd.DataFrame(records)
                 if field == 'building_area_increase':
                     df=df.drop(columns=['pair'])
                 df['bbl']=pd.to_numeric(df['bbl'], downcast='integer')
-                return df,df.shape[0]
+                return df
             else:
-                return 'This is no outlier.'
-        
-        version_pair = v1 + '-' + v2
+                return pd.DataFrame()
+
+        def display_dataframe(v1_outlier, field):
+            df = fetch_dataframe(v1_outlier, field)
+            if df.empty:
+                st.write( "There is no outlier.")
+            else:
+                AgGrid(df)
+                st.write(f'There are {df.shape[0]} outliers in total.')
+
+        version_pair = f"{v1}-{v2}"
```

Comment:
Does it  make more sense to have this set where `outlier_records` and `v1_outlier` are set?